src/steps/iSetFocus.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	docs: {
+		description: "Set focus on a control",
+		synopsis: "I set focus on <controlId> [in <viewName> view]",
+		examples: ["I set focus on inputField", "I set focus on saveButton in Creation view"]
+	},
+	icon: "edit",
+	regexp: new RegExp([
+    "^I set focus on\\s", //start of string
+    "(.+?)", //<controlId>
+    "(\\sin\\s([a-zA-Z0-9\\.]+)\\sview)?", // view part
+    "$" //end of string
+  ].join("")),
+	action: function(sControlId, sViewPart, sViewName) {
+		var that = this;
+		var oWaitForOptions = {
+			id: sControlId,
+			success: function(oControl) {
+
+        that.Opa5.assert.ok(true, "Control " + sControlId + " has been found in " + sViewName + " view");
```

Comment:
Imagine reading the qunit output. Would be best something like: "control XYZ was focussed". It's more descriptive than the step. Perhaps also better to test, after focus() was called, that the control is really focussed. One assert would be fine here "control XYZ was focussed" or "focus was set on XYZ control". If the control is not found, the focus could not be set anyway... one could check if the control can be seen before setting the focus after all.